docs/getting-started.mdx

```diff
@@ -1,81 +1,85 @@
+- AWS [c5.2xlarge](https://www.ec2instances.info/?filter=c5.2xl&region=us-west-2&cost_duration=daily&selected=c5.2xlarge)
+- GCP [c2-standard-8](https://cloud.google.com/compute/docs/compute-optimized-machines)
+- Azure [Standard_F8s_v2](https://learn.microsoft.com/en-us/azure/virtual-machines/fsv2-series)
+- Digital Ocean [c-8-16gib](https://cloud.digitalocean.com/droplets/new?size=c-8-16gib)
 
-## Installation
+Check out our [Connecting to the Mina Testnet (Google Cloud)](https://youtu.be/M9lE2lgyVEo) video for steps to configure your Mina node on Google Cloud.
 
-The newest binary releases can be found below. Instructions are provided for macOS and Linux below:
+## Installation
 
-This is a large download, around 1GB, so the install might take some time.
+Use the installation steps for your environment to install the newest binary releases. The binary download is around 1 GB, so the installation might take a few minutes.
 
-### Ubuntu 18.04 / Debian 9
+### Ubuntu 18.04 and Debian 9
 
-Add the Mina Debian repo and install:
+To add the Mina Debian repo and install the binary:
 
 ```
 echo "deb [trusted=yes] http://packages.o1test.net release main" | sudo tee /etc/apt/sources.list.d/mina.list
 sudo apt-get update
 sudo apt-get install -y curl unzip mina-mainnet=1.1.5-a42bdee
 ```
 
-Check that daemon installed correctly by running `mina version`. The output should read `Commit a42bdeef6b0c15ee34616e4df76c882b0c5c7c2a on branch master`.
+To verify your installation:
+
+- `mina version`
+
+The expected output is `Commit a42bdeef6b0c15ee34616e4df76c882b0c5c7c2a on branch master`.
 
 ### Windows
 
-Windows is not yet natively supported. If you are interested in receiving a grant to develop Mina for Windows, please email grants@o1labs.org or reach out in the [Discord server](https://bit.ly/MinaDiscord).
+Windows is not natively supported. 
 
-Please instead use the Docker instructions below.
+To install Mina on Windows, use the Docker instructions.
 
 ### macOS
 
-macOS is used by developers, but we are not officially supporting any pre-built packages at this time.
-
-Please instead use the Docker instructions below or you may build from source.
+You can install Mina on macOS using the Docker instructions or build from source.
 
 ### Docker
 
-Docker is cross-platform tool that allows you to run a virtual machine that will execute any needed software. Please follow the official installation instructions for your platform on [this page](https://docs.docker.com/get-docker/). Once docker is installed, proceed to the Mina Docker instructions on Connect to the Network
+Docker is a cross-platform tool. You can download and install Docker on multiple platforms. Use the Docker installation instructions for your platform on [Get Docker](https://docs.docker.com/get-docker/). After Docker is installed, follow the instructions on [Connecting to the Network](connecting-to-the-network#docker).
```

Comment:
I might have introduced invalid link syntax here; it's my first time working in .mdx files in this environment.